var/spack/repos/builtin/packages/py-pystache/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPystache(PythonPackage):
+    """Pystache is a Python implementation of Mustache. Mustache is a
+    framework-agnostic, logic-free templating system inspired by ctemplate and
+    et."""
+
+    homepage = "git@github.com:sarnold/pystache.git"
```

Comment:
Since these are more intended for documentation, would prefer IMO a URL using the `https` protocol, e.g.,:

```suggestion
    homepage = "https://github.com/sarnold/pystache"
```

or 

```suggestion
    homepage = "https://pypi.org/project/pystache/"
```